test/extensions/filters/http/adaptive_concurrency/adaptive_concurrency_filter_test.cc

```diff
@@ -30,6 +30,19 @@ class MockConcurrencyController : public ConcurrencyController::ConcurrencyContr
   uint32_t concurrencyLimit() const override { return 0; }
 };
 
+class AdaptiveConcurrencyFilterConfigTest : public testing::Test {
+public:
+  AdaptiveConcurrencyFilterConfigTest() = default;
+
+  envoy::config::filter::http::adaptive_concurrency::v2alpha::AdaptiveConcurrency
```

Comment:
nit: I would just make this a free function above to avoid duplicating it twice.